readme.md

```diff
@@ -44,7 +50,7 @@ unsubscribe();
 
 ## API
 
-### exitHook(onExit)
+### exitHook(onExit, maxWait?)
```

Comment:
This should be an options-object. `exitHook(..., 100)` is not very readable at the callsite.